keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -491,37 +456,23 @@ Defined using feature gate
 - Plaintext output roughly matches explain output
 - OpenAPIV3 (raw json) output implemented
 
-#### Alpha 2
-
-If we decide to move ahead with the `md` and `html` outputs, an Alpha 2 may
-be required.
-
-- `md` output implemented (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Section for each individual GVK
-  - All types hyperlink to specific section
--  basic `html` output  (or dropped from design due to continued debate)
-  - Table of contents all GVKs grouped by Group then Version.
-  - Page for each individual GVK.
-  - All types hyperlink to their specific page
-  - Searchable by name, description, field name.
-
 #### Beta
 
-- kube-openAPI v3 JSON deserialization is optimized to take less than 150ms on
-  most machines
 - OpenAPI V3 is enabled by default on at least one version within kubectl's support window.
 As of Kubernetes 1.24 OpenAPIV3 entered beta and become enabled by default, therefore meeting this requirement.
+In Kubectl for release 1.25, all k8s versions within support window will be able to have OpenAPIV3 enabled.
+However, the fallback is kept around since it may not always be enabled.
+
 - `--output plaintext` is on-by-default and environment variable is removed/on by default
 - `--output plaintext-openapiv2` added as a name for the old `explain` implementation, so the feature may be positively disabled.
```

Comment:
Does this flag value already exist in the current alpha? I'm thinking of the migration path where someone wants to opt out: they need to be able to specify the flag in a version that does not default to the value they don't want so that they can do the upgrade without temporarily using the (for them) bad default.

Reply:
`--output plaintext-openapiv2` is new in beta. If someone is migrating they can specify no `--output` argument which will try both approaches. 

I see the concern for this commands invoked in scripts and automated tools, but for an interactive tool like `explain` while it'd be best to have the flag around beforehand I am thinking it is not a requirement.